batchservice/src/main/java/nu/yona/server/BatchServiceApplication.java

```diff
@@ -34,4 +37,17 @@ public TaskScheduler taskScheduler()
 	{
 		return new ConcurrentTaskScheduler(); // single threaded by default
 	}
+
+	@Bean(name = "noopItemWriter")
+	public ItemWriter<Object> noopItemWriter()
+	{
+		return new ItemWriter<Object>()
+		{
+
+			@Override
+			public void write(List<? extends Object> items) throws Exception
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS1186)